docs/expo_crash_reporting.md

```diff
@@ -0,0 +1,81 @@
+# Expo Crash Reporting and Error Tracking
+
+## Overview
+
+Enable Expo Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring.
+
+With this feature, you can access the following features:
+
+-   Aggregated Expo crash dashboards and attributes
+-   Symbolicated iOS and deobfuscated Android crash reports
+-   Trend analysis with Expo error tracking
+
+In order to symbolicate your stack traces and deobfuscate Android crashes, upload your .dSYM, Proguard mapping files and source maps to Datadog using the `expo-datadog` config plugin.
+
+Your crash reports appear in [**Error Tracking**][1].
+
+## Setup
+
+Use the [`expo-datadog` package and configuration plugin][2]. For more information, see the [Expo and Expo Go documentation][3].
+
+Run `eas secret:create` to set `DATADOG_API_KEY` and `DD_API_KEY` to your Datadog API key.
```

Comment:
We should propose to set only one of them, same change should be done in other places.

```suggestion
Run `eas secret:create` to set `DATADOG_API_KEY` or `DD_API_KEY` to your Datadog API key.
```

Reply:
I'll change it to `DATADOG_API_KEY` once the Gradle plugin accepts this environment variable. Until then we need both of them if we want to cover all mapping files.